### ISSUE TITLE
Store active zoom, rotation, and position

### ISSUE THREAD

**Issue by KenLucke** [Kragrathea/OctoPrint-PrettyGCode#72]
Please consider storing the zoom, location, rotation, etc., along with the prefs.

The issue is, when OctoPrint is reloaded, it loses all this (even during the same print), so I have to re-zoom, re-rotate, and drag the grid back to where I had it before the reload.  A "reset to previous view on reload" pref could be added, so if people did not want to have this happen, they could choose not to do so.

BTW, I was told by the "other" team that there was no way to overcome the browser limitation on displaying gcode for very large files (in the built-in gcode viewer).  Yet, somehow, you seem to have had no problem doing that (at least so far for files up to 420mb that I have tried).  Good for you.

**Comment by lightmaster** [comment]
Yes, this would be awesome. Or at the very least, load up the gcode with it rotated so you look at the front of your print, not the right side.
